### ISSUE TITLE
Error look this please fix it

### ISSUE THREAD

**Issue by QPLAYid** [radimitrov/CSharpShellApp#240]
![Screenshot_2023-02-17-17-15-59-324_com radinc csharpshell](https://user-images.githubusercontent.com/102419809/219589936-93fab7bc-4ec4-4f16-bfc4-fe615552c0da.jpg)

**Comment by QPLAYid** [comment]
This version app : 3.0.7 and 3.0.8 same error

**Comment by radimitrov** [comment]
No, to my knowledge nothing in the app code or dependencies tries to access a "filesconfig.json".
This is your code. In which case the Exception is self descriptive: file (path) doesn't exist. Check your path

**Comment by QPLAYid** [comment]
I try old version ist work but i update and doesnt work

**Comment by QPLAYid** [comment]
![Screenshot_2023-02-17-21-36-11-516_bin mt plus](https://user-images.githubusercontent.com/102419809/219655247-5c8fcbed-6bf8-4f57-aa02-266c6eb5f27d.jpg)


Already fix but must manualy edit file name to
name filesconfig.json

**Comment by QPLAYid** [comment]
![Screenshot_2023-02-17-21-38-05-667_com radinc csharpshell](https://user-images.githubusercontent.com/102419809/219655518-ce8cd926-7418-4603-b4e7-86e60922e9ca.jpg)


And work now 

But firs install and run project this file name not filesconfig.json and must manual edit rename to fileconfig.json

**Comment by radimitrov** [comment]
The APK might not have a writable default path? Are you using relative paths?
However the second statement is impossible unless you've renamed the file from "file**s**config.json" to "fileconfig.json" since paths don't change by themselves.

**Comment by QPLAYid** [comment]
Yes must manualy edit but after reopen app error because apk cant read filesconfig.json

**Comment by QPLAYid** [comment]
![Screenshot_2023-02-19-14-33-26-527-edit_bin mt plus](https://user-images.githubusercontent.com/102419809/219927569-32487db1-71a8-4c77-ae71-6376f14745fd.jpg)


So i created two file 
CSharpShellSettings.json fix for reopen app error
And 
filesconfig.json for fix project error

**Comment by QPLAYid** [comment]
![Screenshot_2023-02-19-14-43-53-129_com companyname program](https://user-images.githubusercontent.com/102419809/219930217-6bf87b7e-d65f-40a6-8395-c8705e65a109.jpg)



Same error after i export to apk 
And i installed apk and open i get same error

**Comment by QPLAYid** [comment]
> The APK might not have a writable default path? Are you using relative paths? However the second statement is impossible unless you've renamed the file from "file**s**config.json" to "fileconfig.json" since paths don't change by themselves.

Can you fix it ? 
Musy manually add file name filesconfig.json

**Comment by QPLAYid** [comment]
It same when i firs time instaal c shell and i run error must create new file on directori root 
/Data/app/cshell 
But versi old can doit without add filels because c shell auto create

**Comment by QPLAYid** [comment]
> @QPLAYid I can't fix since I don't think there is something wrong. I've already released an update to the plugin that does `Directory.SetCurrentDirectory(FilesDir.AbsolutePath);`. So that is the default path. That screenshot looks like C# Shell's app data folder (DataDir). To clarify you do know that when you install an APK it has its own app private files directories, right? Different paths, different access rights - 2 apps can't access each other's private directories. Your files can't automatically get copied into the new app's private directories.

But if cant. Its okay
